style.css

```diff
@@ -0,0 +1,127 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700&display=swap");
+
+:root {
+  --primary-color: #36b37e;
+  --secondary-color: #172b4d;
+  --primary-font: "Poppins", sans-serif;
+  --secondary-font: "Roboto", sans-serif;
+  --tertiary-font: "Inter", sans-serif;
+  --skills-list-bg: #ebf0ee;
+  --skills-list-color: #3a4a42;
+  --p-text-color: #42526e;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.topBar-nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 92px;
+}
+
+/* My Logo */
+
+.logo {
+  padding-left: 10px;
+}
+
+/* Three dash-setting */
+
+.icon-Union {
+  padding-right: 10px;
+}
+
+/* The headline section  */
+
+.content-section {
+  width: 100%;
+  height: 100%;
```

Comment:
- [ ] It's much more better now that your app is responsive, but I observed that in the process of making it responsive all the background-images are not showing as it were before. **You can set height for your parent div i.e. `.content-section:718px` and child element (`.main-container`) take that automatically as `height:100%`**.